linkerd/protocol/h2/src/test/scala/io/buoyant/linkerd/protocol/h2/H2ClassifiersTest.scala

```diff
@@ -167,19 +166,4 @@ class H2StreamClassifiersTest extends FunSuite {
       }
     }
   }
-
-  test("NonRetryableStream: makes RetryableFailures NonRetryableFailures when chunked") {
```

Comment:
NonRetryableStream went away.  We can do retries on streams!  Or we will be able to soon.